aspnetcore/security/docker-https.md

```diff
@@ -52,22 +52,22 @@ Use the following instructions for your operating system configuration.
 Generate certificate and configure local machine:
 
 ```dotnetcli
-dotnet dev-certs https -ep %USERPROFILE%\.aspnet\https\aspnetapp.pfx -p <password>
+dotnet dev-certs https -ep %USERPROFILE%\.aspnet\https\aspnetapp.pfx -p <CREDENTIAL_PLACEHOLDER>
 dotnet dev-certs https --trust
 ```
 
-In the preceding commands, replace `<password>` with a password.
+In the preceding commands, replace `<CREDENTIAL_PLACEHOLDER>` with a password.
 
 Run the container image with ASP.NET Core configured for HTTPS in a command shell:
 
 ```console
-docker pull mcr.microsoft.com/dotnet/samples:aspnetapp
-docker run --rm -it -p 8000:80 -p 8001:443 -e ASPNETCORE_URLS="https://+;http://+" -e ASPNETCORE_HTTPS_PORT=8001 -e ASPNETCORE_Kestrel__Certificates__Default__Password="password" -e ASPNETCORE_Kestrel__Certificates__Default__Path=/https/aspnetapp.pfx -v %USERPROFILE%\.aspnet\https:/https/ mcr.microsoft.com/dotnet/samples:aspnetapp
+docker pull mcr.microsoft.com/dotnet/core/samples:aspnetapp
```

Comment:
There are merge issues with the removal of `core` in the repo path.
`dotnet/core/samples` should all be changed to `dotnet/samples`

Reply:
> There are merge issues with the removal of `core` in the repo path. `dotnet/core/samples` should all be changed to `dotnet/samples`

Fixed with [this commit](https://github.com/dotnet/AspNetCore.Docs/pull/27186/commits/e45fae27ec27d13b81511e9d9022db9a063eee9a)